apps/web/app/features/stacking/user-positions.tsx

```diff
@@ -124,13 +161,33 @@ export function UserPositions({ stacksAddress }: UserPositionsProps) {
           />
         </InfoGrid.Cell>
         <InfoGrid.Cell>
-          <ValueDisplayer name="APR" value="~10%" />
+          <ValueDisplayer name="APR" isLoading={fakeLoading} value="~10%" />
         </InfoGrid.Cell>
         <InfoGrid.Cell>
-          <ValueDisplayer name="Next rewards" value="~10 days" />
+          <ValueDisplayer name="Next rewards" isLoading={fakeLoading} value="~10 days" />
         </InfoGrid.Cell>
         <InfoGrid.Cell>
-          <ValueDisplayer name="Rewards token" value="STX" />
+          <ValueDisplayer
+            name="Rewards token"
+            isLoading={fakeLoading}
+            value={
+              <Flex gap="space.02" alignItems="center">
+                <StacksIcon />
+                STX
+              </Flex>
+            }
+            customSkeletonLayout={
+              <Flex gap="space.02" alignItems="center">
+                <SkeletonLoader
+                  height="24"
+                  width="24"
+                  borderRadius="round"
+                  isLoading={fakeLoading}
+                />
+                <SkeletonLoader height="15" width="80" isLoading={fakeLoading} />
+              </Flex>
```

Comment:
Think I mentioned on another PR, but I'd love to see more of a compositional approach used. If a new requirement comes along, do we continually extend the API of `ValueDisplayer`, or create new components that use it?